### PR TITLE
Adds is_billing_enabled to order object

### LINKED ISSUE #5617: Add isBillingEnabled to order object
**Is your feature request related to a problem? Please describe.**

In redesigning the order review page, we need to know is billing was enabled for that particular order or not
Currently there is no way to know this.

**Describe the solution you'd like**

Adding `isBillingEnabled` field to the `order` can allow us to get this info

**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by shreyanshdwivedi**
Fixes #5617 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
In redesigning the order review page, we need to know is billing was enabled for that particular order or not
Currently there is no way to know this

#### Changes proposed in this pull request:

- Adding `isBillingEnabled` field to the `order` can allow us to get this info

### CODE REVIEW

File: migrations/versions/565ea5bc3937_.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '565ea5bc3937'
+down_revision = '91ee86a38001'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('orders', sa.Column('is_billing_enabled', sa.Boolean(), nullable=True))
```
Comment (CosmicCoder96) [inline migrations/versions/565ea5bc3937_.py]: make server default false, as we can't have this as NULL for past orders where billing info was indeed present.

Comment (iamareebjamal) [inline migrations/versions/565ea5bc3937_.py]: Remove nullable=True

Comment (shreyanshdwivedi) [reply migrations/versions/565ea5bc3937_.py]: Done

Comment (shreyanshdwivedi) [reply migrations/versions/565ea5bc3937_.py]: Done

**Comment by shreyanshdwivedi** [comment]
@CosmicCoder96 @iamareebjamal please review again

File: migrations/versions/565ea5bc3937_.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+# revision identifiers, used by Alembic.
+revision = '565ea5bc3937'
+down_revision = '91ee86a38001'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('orders', sa.Column('is_billing_enabled', sa.Boolean()), server_default='False')
```
Comment (CosmicCoder96) [inline migrations/versions/565ea5bc3937_.py]:
```python
server_default=False
```

Comment (iamareebjamal) [reply migrations/versions/565ea5bc3937_.py]: No, this is correct

Comment (shreyanshdwivedi) [reply migrations/versions/565ea5bc3937_.py]: @iamareebjamal please review

Comment (CosmicCoder96) [reply migrations/versions/565ea5bc3937_.py]: it should have been 
```python
    op.add_column('orders', sa.Column('is_billing_enabled', sa.Boolean(), server_default='False'))
```

Comment (uds5501) [reply migrations/versions/565ea5bc3937_.py]: @CosmicCoder96 @iamareebjamal migration fixed in #5656

**Pull request merged**